src/Websites/Websites/Cmdlets/AppServicePlans/NewAzureAppServicePlan.cs

```diff
@@ -73,9 +73,10 @@ public class NewAzureAppServicePlanCmdlet : AppServicePlanBaseCmdlet
 
         public override void ExecuteCmdlet()
         {
-            if (HyperV.IsPresent && Tier != "PremiumContainer")
+            if (HyperV.IsPresent && 
+                (Tier != "PremiumContainer" && Tier != "PremiumV3"))
```

Comment:
>Tier != "PremiumV3" [](start = 47, length = 19)

should we worry about string case here ?

Reply:
We validate specifically against the values PSArgumentCompleter for Tier (that is the general convention in Azure PowerShell).